lib/plugins/aws/deployFunction.js

```diff
@@ -244,6 +318,7 @@ class AwsDeployFunction {
 
     await this.provider.request('Lambda', 'updateFunctionCode', params);
     this.serverless.cli.log(`Successfully deployed function: ${this.options.function}`);
+    return;
```

Comment:
This change I guess is not necessary (?)

Reply:
that's correct, I was toying with checking if the function was deployed or not and returning here bool value